Dockerfile

```diff
@@ -4,18 +4,31 @@ FROM $IMAGE_BASE:10
 #COPY requires one valid argument, second can be nonexistent
 COPY empty_file tmp/qemu-arm-stati[c] /usr/bin/
 
-RUN apt-get update && apt-get -y install libavahi-compat-libdnssd-dev
+RUN apt-get update && apt-get -y install apt-utils
+RUN apt-get update && apt-get -y install sudo git python3 python build-essential avahi-daemon avahi-discover avahi-utils libnss-mdns mdns-scan libavahi-compat-libdnssd-dev sysstat procps
```

Comment:
There's sudo here still, but not used?

apt-get update is run twice and install is on two rows?

Reply:
I'll change all related files to docker-scripts -folder.

Sudo is still there, but not used in scripts. If you would like to login container and install something, that would require sudo (sudo apt-get ...). If you like that it should be removed totally, then I'll do it for this PR.

1st apt-get install apt-utils and after that starts actual apt-get process. OK?

Reply:
Without sudo `set-system-time` fails `date: cannot set date: Operation not permitted`

Reply:
Plugins that need sudo is a separate issue imho. Even if sudo removes the error from setting system time I'd be surprised that trying to do it from an unprivileged container would succeed all the way.

The reason that I am pestering about this is that I'd like to move in general towards having less privileges by default, not more. Like for example set-system-time could use a separate script that the user can enable setuid on, or some such mechanism.

>1st apt-get install apt-utils and after that starts actual apt-get process. OK?

Yes, I can see what it does , but I'd like to understand why that is needed. Coming back to this after several months that will look odd, so an inline comment as to *why* that is needed would be good.

Reply:
Removed both `RUN apt-get update && apt-get -y install apt-utils` and `sudo`.